source/manual/world-taxonomy.html.md

```diff
@@ -0,0 +1,75 @@
+
+There are a small number of generic taxons that should be displayed on each of the individual world pages, for example [passports and emergency travel documents][passports]. These are then mirrored in the world pages, for example [passports and emergency travel documents Vietnam][vietnamese-passports], which has information specific to that country.
+
+You can view the hierarchy of the world taxonomy in [content-tagger][taxon-list].
+
+[topic-taxonomy]: https://docs.publishing.service.gov.uk/manual/taxonomy.html
+[latvia-doctors]: https://www.gov.uk/government/publications/latvia-list-of-medical-facilities
+[live-latvia]: https://www.gov.uk/world/living-in-latvia
+[schema]: https://github.com/alphagov/govuk-content-schemas
+[passports-generic]: https://www.gov.uk/world/passports-and-emergency-travel-documents
+[taxon-list]: https://content-tagger.publishing.service.gov.uk/taxons/91b8ef20-74e7-4552-880c-50e6d73c2ff9
+[passports]: https://www.gov.uk/world/passports-and-emergency-travel-documents
+[vietnamese-passports]: https://www.gov.uk/world/passports-and-emergency-travel-documents-vietnam
+
+## Differences between topic taxonomy and world taxonomy
+
+Whilst world taxons are treated in the same way as the regular [taxonomy][taxonomy] on a data level, they are a separate entity and do not share the same root. They cannot be found by visiting the root of the GOV.UK taxonomy meaning that they are simultaneously part of and not part of the core taxonomy.
+
+The two have very different structures. The world taxon structure has a breadth of over 200, one for each country and a few generic. The majority of these taxons have only one level below them while the rest have none. The topic taxonomy is an entirely different shape with 21 taxons under the GOV.UK home page each of which with a variable depth up to 5 or 6 levels.
+
+GOV.UK has 3780 published taxons as of the time of writing (03/02/2020), of those 2488 are world taxons meaning that they represent a substantial majority of all taxons.
+
+Unlike the rest of the taxonomy only content associated with [specific organisations][spec-orgs] is allowed to be tagged to world taxons (DFID, FCO, UKVI, DIT, DExEU) and only [particular document types][specific-doc-types] (guidance, forms, detailed guides, and document collections) may be tagged to them. Presumably this is because the world pages are intended to be curation mechanisms for FCO and similar rather than content tagged to a topic.
+
+They are rendered in a completely different format to regular topic taxons. Compare https://www.gov.uk/education/running-and-managing-a-school with https://www.gov.uk/world/yemen.
```

Comment:
"They are rendered in a completely different format to regular topic taxons. Compare" -> "World taxons are rendered differently on the frontend to topic taxons. Compare"

Reply:
👍